spec/controllers/data_values_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe DataValuesController, :type => :controller do
+
+end
```

Comment:
Let's delete specs that we aren't using. (And also since we don't usually write controller specs anyway.)